rero_ils/modules/acq_receipts/api.py

```diff
@@ -181,11 +180,32 @@ def total_amount(self):
 
         :return the receipt total amount rounded on 0.01.
         """
-        # TODO: adapt this method after adding the receipt_lines resource.
-        # total = self._get_receipt_lines_total_amount()
-        total = sum([fee.get('amount') for fee in self.amount_adjustments])
+        # Compute the total of all related receipt line
+        search = AcqReceiptLinesSearch() \
+            .filter('term', acq_receipt__pid=self.pid)
+        search.aggs.metric(
+            'receipt_total_amount',
+            'sum',
+            field='total_amount',
+            script={
+                'source': 'Math.round(_value*100)/100.00'
+            }
+        )
+        results = search.execute()
+        total = results.aggregations.receipt_total_amount.value
+        # Add the sum of all adjustments
+        total += sum(fee.get('amount') for fee in self.amount_adjustments)
         return round(total, 2)
 
+    @property
+    def total_item_quantity(self):
```

Comment:
Do we need this method `total_item_quantity`? I do not see yet used in the code.
will we need to compare this `quantity` with the `order.quantity` one day?

Reply:
It's used into the listener to add the information into ES.
I need it for UI to display the number of item included into a receipt.